internal/checker/terraform/rules/gcp/google_project_iam_binding_service_acc_project_level_not_used.rego

```diff
@@ -0,0 +1,67 @@
+# METADATA
+# title: "Verify that the Default Service Account is not being employed within a project-level context"
+# description: "Confirm that the Default Service Account is not being utilized at the project level to enhance security."
+# scope: package
+# related_resources:
+# - https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/google_project_iam#google_project_iam_member
+# custom:
+#   id: CB_TFGCP_016
+#   severity: CRITICAL
+package lib.terraform.CB_TFGCP_016
+
+import future.keywords.in
+
+isvalid(block) {
+	supported_resources := ["google_project_iam_member", "google_project_iam_binding"]
+	block.Type == "resource"
+	some label in block.Labels
+	label in supported_resources
+}
+
+resource[resource] {
+	some block in pass
+	resource := concat(".", block.Labels)
+}
+
+resource[resource] {
+	some block in fail
+	resource := concat(".", block.Labels)
+}
+
```

Comment:
you can swap pass and fail logic on this one, then removed line 41-45

fail[block] {
	some block in input
	isvalid(block)
	invalid_member := [
		"project-number-compute@developer.gserviceaccount.com",
		"project-id@appspot.gserviceaccount.com",
	]
        block.Attributes.member in invalid_member
}